docs/content/contributing/bicep/bicep-contribution-flow/owner-contribution-flow.md

```diff
@@ -190,3 +165,7 @@ This checklist can be used by anyone (author/contributor/owner) developing AVM B
 6. [Createupdate-and-run-tests](https://azure.github.io/Azure-Verified-Modules/contributing/bicep/bicep-contribution-flow/#5-createupdate-and-run-tests)
 
 7. Create PR and reference the status badge of your pipeline run see [here](https://azure.github.io/Azure-Verified-Modules/contributing/bicep/bicep-contribution-flow/#6-create-a-pull-request-to-the-public-bicep-registry).
+
+8. After a pull request has been created it is important to update the [AVM issue](https://github.com/Azure/Azure-Verified-Modules/issues) assocaited with your module, with a link to the pull request you created in BRM and mention the **@Azure/avm-core-team**.
```

Comment:
The correct team to mention is `Azure/avm-core-team-technical-bicep`. 

I think the text should say: 
"... it is important to update the [AVM module proposal](https://aka.ms/AVM/ModuleProposals) issue associated with your module, with a link to the pull request you created in BRM and mention the person who helped triage your module or the `@Azure/avm-core-team-technical-bicep` team."